src/utils/commons.ts

```diff
@@ -83,3 +83,16 @@ export const deepClone = <T>(obj: T): T => {
 
   return clone;
 };
+
+export const convertStringToBoolean = (str: string): boolean | null => {
```

Comment:
아래 내용에 의하면 리턴 타입이 하나 더 추가되어야겠네영

```suggestion
export const convertStringToBoolean = (str: string): boolean | null | undefined => {
```

Reply:
TRUE, FALSE, NULL 도 undefined 로 되는 것도 의도하신 바가 맞을까요 ?

Reply:
확인해보니 반환한 값이 undefined인지 확인하는 경우가 없어서 undefined를 제거하고 boolean과 null 값만 반환하도록 수정했습니다. d301f5f